src/extensibility/dotnet-extensibility/DataModel/ResourceCatalog.cs

```diff
@@ -39,12 +39,11 @@ public partial record ResourceCatalog
     /// <summary>
     /// Gets a regular expression to validate a resource catalog identifier.
     /// </summary>
-    [JsonIgnore]
-#warning Remove this when https://github.com/RicoSuter/NSwag/issues/4681 is solved
     public static Regex ValidIdExpression { get; } = ValidIdExpressionRegex();
 
-    [JsonIgnore]
-#warning Remove this when https://github.com/RicoSuter/NSwag/issues/4681 is solved
+    /// <summary>
```

Comment:
This doc comment is not strictly necessary since the property is private (and there are not doc comments for private and internal properties anywhere else in the project)